deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/conf/layers/BaseRecurrentLayer.java

```diff
@@ -42,10 +52,33 @@ public void setNIn(InputType inputType, boolean override) {
     public InputPreProcessor getPreProcessorForInputType(InputType inputType) {
         return InputTypeUtil.getPreprocessorForInputTypeRnnLayers(inputType, getLayerName());
     }
-
+    //class Builder<T extends FeedForwardLayer.Builder<T>> extends Layer.Builder<T>
 
     @AllArgsConstructor
-    public static abstract class Builder<T extends Builder<T>> extends FeedForwardLayer.Builder<T> {
+    public static abstract class Builder<T extends BaseRecurrentLayer.Builder<T>> extends FeedForwardLayer.Builder<T> {
+        protected boolean useLayerNormalization = false;
+
+
+        public Builder(Builder<T> builder) {
```

Comment:
I don't see why we need these sorts of constructors... We've never needed them before, and there's nothing special about the useLayerNormalization arg (it's just another  boolean)... unless I'm missing something?